packages/xod-client/src/editor/selectors.js

```diff
@@ -52,6 +52,29 @@ export const getCurrentPatchOffset = createSelector(
   R.propOr(DEFAULT_PANNING_OFFSET, 'offset')
 );
 
+// :: State -> EditorTabs
+export const getPreparedTabs = createSelector(
```

Comment:
How is this related to the bump?

Please, review a PR by yourself as a first step before posting the PR

Reply:
Just a little refactoring and optimization.
Some other functions are fixed to work properly with turned on checkTypes.